frontend/src/components/LogoImage.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div>
+    <v-img src="../assets/logo.png" alt="logo" class="img"></v-img>
```

Comment:
will be better if we renamed this to auth_logo 
We'll have a logo for the project in the future